pkg/chains/storage/docdb/docdb.go

```diff
@@ -262,6 +266,18 @@ func populateMongoServerURL(ctx context.Context, cfg config.Config) error {
 	logger := logging.FromContext(ctx)
 	mongoEnv := "MONGO_SERVER_URL"
```

Comment:
looks like we need MONGO_SERVER_URL to put as constant, because it is repeated  in some places